test/tracer.spec.ts

```diff
@@ -489,6 +491,274 @@ describe('Autoblocks Tracer', () => {
+              score: 0.9,
+              threshold: undefined,
+              metadata: undefined,
+              evaluatorExternalId: 'my-evaluator',
+            },
+            {
+              id: mockUuid,
+              score: 0.9,
+              threshold: undefined,
+              metadata: undefined,
+              evaluatorExternalId: 'my-async-evaluator',
+            },
+            {
+              id: mockUuid,
+              score: 0.9,
+              threshold: {
+                gt: 0,
+                lte: 1,
+              },
+              metadata: {
+                'some-metadata-key': 'some-metadata-value',
+              },
+              evaluatorExternalId: 'my-full-info-evaluator',
+            },
+            {
+              id: mockUuid,
+              score: 0.9,
+              threshold: {
+                gt: 0,
+                lte: 1,
+              },
+              metadata: {
+                'some-metadata-key': 'some-metadata-value',
+              },
+              evaluatorExternalId: 'my-async-full-info-evaluator',
+            },
+          ],
+        },
+      });
+    });
+
+    describe('errors', () => {
+      it('does not block if there is an evaluator error', async () => {
+        class ErrorEvaluator extends BaseEventEvaluator {
+          id = 'error-evaluator';
+
+          evaluateEvent(): Evaluation {
+            throw new Error('Something unexpected happened');
+            return {
+              score: 0.9,
+            };
+          }
+        }
+        const tracer = new AutoblocksTracer('mock-ingestion-key');
+        const { traceId } = await tracer.sendEvent('mock-message', {
+          evaluators: [new ErrorEvaluator()],
+        });
+
+        expect(traceId).toEqual('mock-trace-id');
+
+        expectPostRequest({
+          message: 'mock-message',
+          traceId: undefined,
+          timestamp,
+          properties: {},
+        });
+      });
+
+      it('does not block if there is an error in our code running the evaluation', async () => {
+        class ErrorEvaluator extends BaseEventEvaluator {
+          id = 'error-evaluator';
+
+          evaluateEvent(): Evaluation {
+            throw new Error('Something unexpected happened');
+          }
+        }
+        const tracer = new AutoblocksTracer('mock-ingestion-key');
+        const spy = jest
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          .spyOn(AutoblocksTracer.prototype as any, 'runEvaluatorUnsafe')
+          .mockImplementationOnce(() => {
+            throw Error('Brutal!');
+          });
+        const { traceId } = await tracer.sendEvent('mock-message', {
+          evaluators: [new ErrorEvaluator()],
+        });
+
+        expect(traceId).toEqual('mock-trace-id');
+
+        expectPostRequest({
+          message: 'mock-message',
+          traceId: undefined,
+          timestamp,
+          properties: {},
+        });
+
+        spy.mockClear();
```

Comment:
Nit: move this into a beforeEach / after each so that it's guaranteed to be called. As is this won't get called if any of the assertions above fails